### ISSUE TITLE
Interactive Tasks unsets the taskGroupId and dependencies list

### ISSUE THREAD

**Issue by pyoor** [taskcluster/taskcluster#3497]
**Describe the bug**
When creating an interactive task, the taskGroupId and dependency list of the newly created task is different from the original task.

**To Reproduce**

1. Create interactive task
2. View task definition

**Expected behavior**
If this is intended behavior, the modal displayed when creating the interactive task should include wording that describes this behavior.

**Taskcluster version**
37.1.0

**Comment by djmitche** [comment]
@petemoore what do you think -- should we take a look at all the places in the UI that we drop task fields and have a rethink?  I think that's
 * interactive tasks
 * retriggers
 * edit task
but I may be missing one or two cases.

**Comment by petemoore** [comment]
@djmitche Yes, I definitely think so. I don't think the task explorer should mutate the task in any way, I think it should be like a posix fork, and it is up to the user to decide what they want to modify. Thanks!
